src/test/java/de/tum/in/www1/artemis/lecture/AttachmentUnitsIntegrationTest.java

```diff
@@ -78,34 +78,36 @@ void testAll_InstructorNotInCourse_shouldReturnForbidden() throws Exception {
         this.testAllPreAuthorize();
     }
 
-    @Test
-    @WithMockUser(username = TEST_PREFIX + "instructor1", roles = "INSTRUCTOR")
-    void splitLectureFile_asInstructor_shouldGetUnitsInformation() throws Exception {
-        // TODO: move code of the next 3 lines into RequestUtilService and simply invoke request.postWithMultipartFile() --> potentially create a customized version
```

Comment:
We should still add a method in the RequestUtilService, since we will probably also use MultipartFiles in other tests as well